wiki/Client/Interface/en.md

```diff
@@ -87,15 +83,15 @@ Before continuing on, this screen has too many elements to note with easily, not
 
 ![](img/beatmap-metadata.jpg)
 
-This area displays **information on the beatmap difficulty currently selected.** By default, the beatmap whose song is heard in the osu! jukebox is selected when entering the selection screen. In the top left is the ranked status of the beatmap. The title is next. Normally, the romanised title is shown, but if you select `Prefer metadata in original language` in the [Options](/wiki/Client/Options), it will show the Unicode title; this is shown in the upper picture. The beatmapper is also shown, and beatmap information is shown below. From left to right, the values are as follows:
+This area displays **information on the beatmap difficulty currently selected.** By default, the beatmap whose song is heard in the osu! jukebox is selected when entering the selection screen. In the top left is the ranked status of the beatmap. The title is next. Normally, the romanised title is shown, but if you select `Prefer metadata in original language` in the [Options](/wiki/Client/Options), it will show the Unicode title. This is shown in the upper picture. The mapper is also shown, and beatmap information is shown below. From left to right, the values are as follows:
 
 - **Length**: The total length of the beatmap, from start to finish and including breaks. Not to be confused with [drain time](/wiki/Gameplay/Drain_time).
-- **BPM**: The BPM of the beatmap. If (like in the lower picture) there are two BPMS and one in parentheses, this means that the BPM changes throughout the song. It shows the slowest and fastest BPMs, and the value in parentheses is the BPM at the start of the beatmap.
+- **BPM**: The BPM of the beatmap. If (like in the lower picture) there are two BPM and one in parentheses, this means that the BPM changes throughout the song. It shows the slowest and fastest BPM, and the value in parentheses is the BPM at the start of the beatmap.
 - **Objects**: The total amount of [hit objects](/wiki/Hit_object) in the beatmap.
 - **Circles**: The total amount of hit circles in the beatmap.
 - **Sliders**: The total amount of sliders in the beatmap.
 - **Spinners**: The total amount of spinners in the beatmap.
```

Comment:
This mostly checks out. One slight distinction in the "spinners" case is that the count in that case is the count of banana *showers*, i.e. the individual uninterrupted sequences of bananas falling, rather than the count of all individual bananas. But I'm not sure if that term functions on wiki (it definitely functions internally in lazer).

Reply:
yeah we should just use banana showers if that's what the thing is counting - edited to match

Reply:
missing `(osu!catch)` for the juice streams in your suggestion